### ISSUE TITLE
cant load the gem

### ISSUE THREAD

**Issue by zenvdeluca** [sumoheavy/jira-ruby#121]
2.1.5 (main):0 > require 'jira'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jira-0.1.0/lib/jira/cli.rb:28: warning: else without rescue is useless
SyntaxError: /usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jira-0.1.0/lib/jira/cli.rb:24: syntax error, unexpected ':', expecting keyword_then or ',' or ';' or '\n'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/jira-0.1.0/lib/jira/cli.rb:65: syntax error, unexpected keyword_end, expecting end-of-input
from /usr/local/var/rbenv/versions/2.1.5/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'

**Comment by dlanner** [comment]
I am also getting the same error and stack trace:

```
$ ruby -r jira -e "puts 'test'"
~/.rvm/gems/ruby-2.2.0/gems/jira-0.1.0/lib/jira/cli.rb:28: warning: else without rescue is useless
~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require': ~/.rvm/gems/ruby-2.2.0/gems/jira-0.1.0/lib/jira/cli.rb:24: syntax error, unexpected ':', expecting keyword_then or ',' or ';' or '\n' (SyntaxError)
~/.rvm/gems/ruby-2.2.0/gems/jira-0.1.0/lib/jira/cli.rb:65: syntax error, unexpected keyword_end, expecting end-of-input
  from ~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
  from ~/.rvm/gems/ruby-2.2.0/gems/jira-0.1.0/lib/jira.rb:5:in `<top (required)>'
  from ~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require'
  from ~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
  from ~/.rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
```

**Comment by florinpatrascu** [comment]
You may have multiple ruby versions installed and a conflicting path, when you're trying to run your test.

Try this, in your `Gemfile`:
```
gem 'jira-ruby', :require => 'jira'  
```
and then run: `bundle exec ruby your_test_file.rb`

Continue to use: `require 'lira'`, not `jira-ruby`, in your test file.

HTH

**Comment by xamebax** [comment]
It's confusing, got me some time to figure it out, too. What I did after getting those errors:

1. Uninstalled the 'jira' gem using `gem uninstall`.
2. Removed the Gemfile.lock file.
3. Added this line to my Gemfile: `gem 'jira-ruby'`
4. Added this line to my main app.rb file: `require "jira"`
5. ran `bundle install`.

And it worked. There's a gem called 'jira' and a gem called 'jira-ruby' and that where the confusion comes from, I guess.
